.gitignore

```diff
@@ -19,3 +19,4 @@ id_rsa.pub
 known_hosts
 id_rs*
 *ssh*
+packages/**/package-lock.json
```

Comment:
@BeniCheni, this was suggested by @sdalonzo because lerna manages all the deps at the root of the project, so we don't need lockfiles for each package.

Reply:
@unscsprt / @sdalonzo , so IMO, semantic looking at the code base now, one may be confused even why we need a lock file at root to baseline non packages metadata. Why not ignore all lock files from root down?